requirements.txt

```diff
@@ -9,7 +10,7 @@ debtcollector==0.8.0
 decorator==4.0.4
 dj-static==0.0.6
 Django==1.8.5
-django-angular==0.7.15
+-e git+https://github.com/open-craft/django-angular.git@add307dbaa5df05f5839361ba899d88e83d658b2#egg=django-angular
```

Comment:
You can use 0.8.2 from https://github.com/jrief/django-angular/pull/256 now, right?